include/appfwk/detail/DAQModule.hxx

```diff
@@ -0,0 +1,16 @@
+namespace dunedaq::appfwk {
+
+template <typename Child>
+void
```

Comment:
Do we need the template? If we move the bind to the register_command call in the DAQModule implementation, then this can just use a std::functional argument, as in ThreadHelper.

Reply:
Alas yes, otherwise the compiler categorically refuses to assign the bound function to the `std::function` object.
This is the first thing I tried actually, and costed me a couple of hours of sleep...